### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Daniielmayo**

### CODE REVIEW

File: .env
Comment (DanielCalleCO) [inline .env:1]: I thinks this file should not be uploaded. What do you think @abdulflorez ?

File: components/atoms/icons/errorRegisterIcons/ErrorRegisterIcon.module.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.error-register-icon {
+  color: var(--darkGray);
+  width: 1.3rem;
+  margin: 0 0.6rem;
+  background: var(--white);
+  border-radius: 50%;
+  border: 1px solid var(--darkGray);
```
Comment (DanielCalleCO) [inline components/atoms/icons/errorRegisterIcons/ErrorRegisterIcon.module.css:7]: px to rem

File: components/organisms/ChangePasswordUser/ChangePasswordUser.module.css [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+  align-items: center;
+}
+
+.container-arrow-back {
+  width: 100%;
+}
+.link-backProfile {
+  display: flex;
+  align-items: center;
+  margin-bottom: 30px;
```
Comment (DanielCalleCO) [inline components/organisms/ChangePasswordUser/ChangePasswordUser.module.css:31]: px to rem

File: components/organisms/ChangePasswordUser/ChangePasswordUser.module.css [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+  gap: 30px;
```
Comment (DanielCalleCO) [inline components/organisms/ChangePasswordUser/ChangePasswordUser.module.css:32]: px to rem

File: components/organisms/Modal/Modal.module.css [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  padding: 2.4rem;
+}
+
+.modalEditProfile-container {
+  position: relative;
+  width: 23rem;
+  padding: 1.2rem;
+  background: var(--white);
+  border-radius: 0.8rem;
+  box-shadow: #64646f33 0px 7px 29px 0px;
```
Comment (DanielCalleCO) [inline components/organisms/Modal/Modal.module.css:20]: px to rem

File: components/organisms/Modal/Modal.module.css [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+.modalEditProfile-iconClose:hover {
+  border-radius: 0.5rem;
+  color: var(--turquoise);
+  background-color: var(--transparentBackground);
+}
+@media (width >= 1024px) {
+  .modalEditProfile-container {
+    width: 600px;
```
Comment (DanielCalleCO) [inline components/organisms/Modal/Modal.module.css:47]: Not sure, but try to change px by rem here too

File: components/organisms/Modal/Modal.module.css [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+  transition: 0.3s ease all;
+  color: var(--darkBlue);
+}
+
+.modalEditProfile-iconClose:hover {
+  border-radius: 0.5rem;
+  color: var(--turquoise);
+  background-color: var(--transparentBackground);
+}
+@media (width >= 1024px) {
```
Comment (DanielCalleCO) [inline components/organisms/Modal/Modal.module.css:45]: DO you think we can change px by rem here? Or is it better to stand with px due to it is a screen width?

File: src/app/(styles)/404.module.css [hunk 7]
```diff
@@ -0,0 +1,44 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+  text-align: center;
+}
+.icon-404 {
+  width: 250px;
```
Comment (DanielCalleCO) [inline src/app/(styles)/404.module.css:10]: px to rem

File: src/app/(styles)/404.module.css [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+  height: 200px;
```
Comment (DanielCalleCO) [inline src/app/(styles)/404.module.css:11]: px to rem

File: src/app/(styles)/404.module.css [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+  transition: color 0.2s;
+}
+
+.link-home_notFound:hover {
+  color: var(--turquoise);
+}
+
+@media (width >= 720px) {
+  .icon-404 {
+    width: 380px;
```
Comment (DanielCalleCO) [inline src/app/(styles)/404.module.css:38]: px to rem

File: src/app/(styles)/404.module.css [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+    height: 380px;
```
Comment (DanielCalleCO) [inline src/app/(styles)/404.module.css:39]: px to rem

File: src/app/(styles)/404.module.css [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+  text-decoration: none;
+  font-weight: 600;
+  transition: color 0.2s;
+}
+
+.link-home_notFound:hover {
+  color: var(--turquoise);
+}
+
+@media (width >= 720px) {
```
Comment (DanielCalleCO) [inline src/app/(styles)/404.module.css:36]: think about this px here...

File: src/app/(styles)/404.module.css [hunk 12]
```diff
@@ -0,0 +1,44 @@
...
+  .icon-404 {
+    width: 380px;
+    height: 380px;
+  }
+  /* .continer-text-notFound {
```
Comment (DanielCalleCO) [inline src/app/(styles)/404.module.css:41]: Remove this comments

**Review by DanielCalleCO — CHANGES_REQUESTED** [review]

File: .env
Comment (abdulflorez) [reply .env:1]: That's right, but We need a new task to create the workflow that creates the .env file automatically.
I'll create the task for this sprint but that should be included in another PR

**Pull request closed without merging**